<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgArtisticColorSelector</class>
 <widget class="QWidget" name="wdgArtisticColorSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>294</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="KisPopupButton" name="bnColorPrefs">
       <property name="text">
        <string>Pref.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bnReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bnAbsLight">
       <property name="text">
        <string>Abs.</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="KisColorSelector" name="colorSelector">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisPopupButton</class>
   <extends>QPushButton</extends>
   <header location="global">kis_popup_button.h</header>
  </customwidget>
  <customwidget>
   <class>KisColorSelector</class>
   <extends>QWidget</extends>
   <header>kis_color_selector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
